contracts/examples/IdentityExample.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.16;
+pragma abicoder v2;
+
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "../interfaces/IState.sol";
+import "../lib/ClaimBuilder.sol";
+import "../lib/OnChainIdentity.sol";
+import "../lib/IdentityBase.sol";
+
+// /**
+//  * @dev Contract managing onchain identity
+//  */
+contract IdentityExample is IdentityBase {
+
+   function getMaxSmtDepth() public pure override returns (uint256) {
+        return 42;
```

Comment:
Why 42 but not 40?

Reply:
it's just to show that we can override depth (default value - 40)